### DISCUSSION TITLE
Deleting a private repository in an organization that I was working on but left

### DISCUSSION THREAD

**Discussion by ienov** [community/community#22043] [New to GitHub]
<p>Hello, maybe a stupid question, but I did not find an answer to it.<br>
In general, I worked on one project, its private repository is in the organization, but I left there.<br>
I would like them to remove it from there, but just like that, my request will be denied.<br>
Can I request that it be removed?<br>
All the development on it was carried out exclusively by me, all the changes were made by me.</p>
<p>Sorry, my English is bad and I used a translator.</p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/ienov">@ienov</a> that is a difficult one as “private” is a visibility setting not an ownership one.  If it is a repository owned by the organization rather than your personal account you will need to approach the organization owner and try to persuade them to delete.</p>
